dotnet-grad/Controllers/UserController.cs

```diff
@@ -0,0 +1,49 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+using dotnet_grad.Models;
+using Microsoft.AspNetCore.Mvc;
+
+namespace dotnet_grad.Controllers
+{
+  [ApiController]
+  [Route("api/[controller]")]
+  public class UserController : ControllerBase
+  {
+    TestContext testContext;
+    public UserController(TestContext _testContext)
+    {
+      testContext = _testContext;
+    }
+
+    [HttpGet("GetAllUsers")]
```

Comment:
You do not have to declare a specific route for fetching all addresses. It's generally inferred that get without parameters return all, `get?id={}` or `get/{id}` return as specific,`post` creates,`put` and `patch` updates or replaces respectively.